### ISSUE TITLE
Does it support pretrained pytorch weights?

### ISSUE THREAD

**Issue by appledora** [xduan7/hat-cl#1]
Hello, 
Thanks for creating this very neat wrapper. 
I was trying to figure out whether this package supports pretrained weights for models like Resnet and ViT. I have set `pretrained = True` in the timm_models module for resnet, but that doesn't seem to be affecting any of the model architectures.

**Comment by xduan7** [comment]
Thanks for trying out hat-cl. I have updated a new version 0.1.1, which contains the pretrained weight loading for `HATResNet18` and `HATResNet34`. Please note that the attention masks, if randomly initialized, will change the output of the network. You could set `init_strat="dense"` in `HATConfig` so that all attention masks are set to `1` to utilize the pretrained weights. Please refer to [mask_initialization_and_scaling.ipynb](https://github.com/xduan7/hat-cl/blob/scratch/examples/mask_initialization_and_scaling.ipynb) for more details on the initialization.

**Comment by appledora** [comment]
Thank you for the kind response. I will look into the notebooks
